src/main/java/address/controller/RootLayoutController.java

```diff
@@ -82,7 +75,7 @@ private FileChooser getXmlFileChooser() {
     @FXML
     private void handleNew() {
         logger.debug("Wiping current model data.");
-        PrefsManager.getInstance().clearSaveLocation();
+        modelManager.clearPrefsSaveLocation();
```

Comment:
Not related to this PR, but why the save location is resest here? The purpose of this menu option is to wipeout the data, not the location where it is saved, right?